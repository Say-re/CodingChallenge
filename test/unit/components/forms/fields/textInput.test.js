import React from 'react';
import { shallow } from 'enzyme';
import TextInputComponent from '../../../../../src/components/forms/fields/textInput';

describe('Text Input Component Tests', () => {
  test('Shallow TextInputComponent Snapshot', () => {
    const wrapper = shallow(<TextInputComponent />)
    expect(wrapper).toMatchSnapshot();
  })
})