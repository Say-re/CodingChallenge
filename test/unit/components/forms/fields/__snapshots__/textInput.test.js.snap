// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text Input Component Tests Shallow TextInputComponent Snapshot 1`] = `
<input
  onChange={[Function]}
  required={true}
/>
`;
